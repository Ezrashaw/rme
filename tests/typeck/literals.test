0;
6;
6.1;
902.01;
true;
false;