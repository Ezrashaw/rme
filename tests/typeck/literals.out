[33;3m0[0m[90;1m: [0mfloat
[33;3m6[0m[90;1m: [0mfloat
[33;3m6.1[0m[90;1m: [0mfloat
[33;3m902.01[0m[90;1m: [0mfloat
[36;1mtrue[0m[90;1m: [0mbool
[36;1mfalse[0m[90;1m: [0mbool
